### PR TITLE
Minor improvements to Diversity Survey dashboard.

### PR DESCRIPTION
**Pull request by ashercodeorg**
- Tracks student counts for respondents, FARM question respondents, and ethnicity question respondents separately.
- Truncates decimal counts.
- Adds better explanation of numbers to the user.

image

### CODE REVIEW

File: dashboard/app/controllers/admin_reports_controller.rb [hunk 1]
```diff
@@ -40,29 +43,40 @@ def diversity_survey
         next if survey_result.properties.blank?
         @respondents += 1
 
+        # The number of born students associated with the teacher.
```
Comment (breville) [inline dashboard/app/controllers/admin_reports_controller.rb:46]: What does born mean?

**Comment by breville** [comment]
lgtm.  maybe commas for big numbers too?

File: dashboard/app/controllers/admin_reports_controller.rb (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/app/controllers/admin_reports_controller.rb:46]: The user has signed into their account (note that [very] roughly a third of our accounts are unborn.).

**Comment by ashercodeorg** [comment]
Good idea (commas for big numbers). Let me add that to the PR.

**Pull request merged**